plugins/defaultplugins/l3plugin/data_resync.go

```diff
@@ -67,8 +72,9 @@ func (plugin *RouteConfigurator) Resync(nbRoutes []*l3.StaticRoutes_Route) error
 			routeID := routeIdentifier(nbRoute.VrfId, nbRoute.DstIpAddr, nbRoute.NextHopAddr)
 			plugin.RouteIndexes.RegisterName(routeID, plugin.RouteIndexSeq, nbRoute)
 			plugin.RouteIndexSeq++
+			plugin.Log.Debugf("RESYNC routes: route %s registered without additional changes", routeID)
+			break VppRouteIteration
```

Comment:
Isn't simple break enough here, since the `VppRouteIteration` is just before the inner loop.